iree/compiler/Dialect/Flow/IR/FlowOpUtils.h

```diff
@@ -0,0 +1,68 @@
+// Copyright 2020 Google LLC
```

Comment:
These look pretty handy for other dialects too. Since they aren't flow-specific could you throw them in iree/compiler/Utils/OpUtils.h?